README.md

```diff
@@ -24,3 +43,165 @@
 - `gnosis-testnet` block explorer contract verification API does not work.
 
 - `rsk` and `rsk-testnet` block explorers do not have contract verification APIs.
+
+## API
+
+The following variables/functions are exported from this package
+
+### CHAINS
+
+The single source of truth for the list of supported chains. A static array of `Chain` objects.
+
+```ts
+import { CHAINS } from '@api3/chains';
+console.log(CHAINS);
+/* 
+[
+  {
+    name: 'Arbitrum testnet',
+    alias: 'arbitrum-goerli-testnet',
+    id: '421613',
+    ...
+  },
+  ...
+]   
+*/
+```
+
+### getChainByAlias
+
+Returns a single `Chain` record found by it's alias. Throws an error if the `Chain` is not found.
+
+```ts
+import { getChainByAlias } from '@api3/chains';
+console.log(getChainByAlias('ethereum'));
+/*
+{
+  "name": "Ethereum",
+  "alias": "ethereum",
+  "id": "1",
+  ...
+}
+*/
+```
+
+### hardhatConfigNetworks
+
+Returns an object where the key is each chain's alias and the value is an object that can be used with [hardhat](https://hardhat.org).
+
+```ts
+import { hardhatConfigNetworks } from '@api3/chains';
+console.log(hardhatConfigNetworks());
+/*
+{
+  "arbitrum-goerli-testnet": {
+      accounts: { mnemonic: '' },
+      chainId: '421613',
+      url: 'https://...',
+  },
+  ...
+}
+*/
+```
+
+### hardhatEtherscan
+
+Returns an object that can be used with Etherscan.
```

Comment:
```suggestion
Returns an object where the key is each chain's alias and the value is an object that can be used as the `etherscan` field of [`hardhat.config.js`](https://hardhat.org/hardhat-runner/docs/config) (requires the [`hardhat-etherscan` plugin](https://hardhat.org/hardhat-runner/plugins/nomiclabs-hardhat-etherscan)).
```

Reply:
Thanks. I wasn't sure what these functions were actually for 👍🏻